Unit.cpp

```diff
@@ -36,4 +46,47 @@ void Unit::SufferDamage(int damageRecieved)
     if(health < 0) {
         health = 0;
     }
+}
+
+Unit Unit::parseUnit(const string fileName) 
+{
+    vector<string> text;
+    string currentLine;
+
+    ifstream file;
+
+    file.open(fileName);
+
+    if(file) {
+        while (getline(file, currentLine))
+        {
+            text.push_back(currentLine);
+        }
+
+        text.erase(text.end());
+        text.erase(text.begin());
+
+        for (size_t i = 0; i < text.size(); i++)
+        {
+            text[i].erase(remove(text[i].begin(), text[i].end(), ' '), text[i].end());
```

Comment:
most nincs a fejemben az std api, de ezt tuti nem lehet valami rovidebb modon? Illetve ha mar ennyiszer kopizva van, akar egy fuggvenybe is ki lehetne akkor szervezni, ha muszaj ilyen hosszan.

Reply:
@csibe1999 kiszervezte külön függvénybe, így egy picit átláthatóbb és letisztultabb.